signinapp/events.py

```diff
@@ -266,3 +293,48 @@ def search():
         return render_template("search/events.html.jinja2", form=form, results=results)
 
     return render_template("search/hours.html.jinja2", form=form, results=None)
+
+
+@events.route("/events/register", methods=["GET", "POST"])
+def register_event():
+    event: List[Event] = db.session.get(Event, request.args["event_id"])
```

Comment:
You don't need `from typing import List`, you can use `list` directly as of [Python 3.9](https://docs.python.org/3/whatsnew/3.9.html)